### DISCUSSION TITLE
A few things to make the website less "sketchy"

### DISCUSSION THREAD

**Discussion by teknowledgist** [goastian/midori-desktop#59] [Q&A]
Open Source projects are successful if folks use them.  Folks use them if they feel they can trust them.  [The website](https://astian.org/midori-browser/) (and the repo) has some features that made me suspicious.

1. "About us" and "Contact us" don't go anywhere.
2. The phone number, "123-456-7890" is clearly fake.
3. Lots of claims about being open source and safe, but not a single link to the source repository.  
4. No link or mention of the license anywhere.  In fact, looking through the source files, it doesn't appear that the complex license page is visible at all without installing Midori first.  Users need to see the license *before* they install the software, or they just won't install it at all.
5. "Official Support" links in the repo get a 404.

I'm sure there are additional "suspicious" things that could be corrected, but that is a start.  All of those should be pretty easy to mitigate.  If getting a support forum up and running is a challenge, just having a "we're working on it" page in the meantime would be better than a 404.  

Thanks!

**Comment by Ponchale** [comment]
We are improving and there are some "jokes" such as the telephone is a joke, and we are in the process of improvement and very soon there will be a visual redesign of everything.

thank you.

**Comment by teknowledgist** [comment]
Ah!  Sorry, humor can be tricky.  I would have made it 867-5309, and have it link to a message like, "We're developing open source software here. If corporations making many billions each year won't let you talk to a human, ain't nobody gonna' be answering that fake number."

Anyway, the most troubling part to me is the inability to find the license or source code from the user-friendly pages.  Showing that off with pride is practically a foundational principle in the OSS universe, and it would be very, very easy to insert links in the existing text.

**Comment by Ponchale** [comment]
Well normally we add the information in the FAQ but if you are absolutely right we must improve and we will continue that, we are in the process of a massive update, so we will change that and mention more our beloved position towards OSS
